packages/web/src/components/track/desktop/ConnectedPlaylistTile.tsx

```diff
@@ -89,41 +87,77 @@ type OwnProps = {
   source?: ModalSource
 }
 
-type ConnectedPlaylistTileProps = OwnProps &
-  ReturnType<typeof mapStateToProps> &
-  ReturnType<typeof mapDispatchToProps>
-
 const ConnectedPlaylistTile = ({
+  uid,
+  id: collectionId,
   ordered,
   index,
   size,
-  collection,
-  userHandle,
   containerClassName,
-  user,
-  tracks,
   togglePlay,
   playTrack,
   pauseTrack,
-  playingUid,
-  isBuffering,
-  isPlaying,
-  goToRoute,
-  record,
   playingTrackId,
   isLoading,
   numLoadingSkeletonRows,
   isUploading,
   hasLoaded,
-  shareCollection,
-  repostCollection,
-  undoRepostCollection,
-  saveCollection,
-  unsaveCollection,
   isTrending,
   isFeed = false,
   source
-}: ConnectedPlaylistTileProps) => {
+}: PlaylistTileProps) => {
+  const dispatch = useDispatch()
+
+  const { data: collection } = useCollection(collectionId)
+  const tracks = useCollectionTracksWithUid(collection, uid)
+  const { data: user } = useUser(collection?.playlist_owner_id)
+  const { handle: userHandle } = user ?? {}
```

Comment:
do we only need the handle? if so, just select it directly

I've been doing
`const { data: handle } = useUser(...)`